setup.py

```diff
@@ -12,8 +12,9 @@ def get_dist(pkgname):
     'numpy',
     'regex',
     'tqdm',
-    'gym'
-
+    'gym',
+    'matplotlib',
+    'opencv-python'
```

Comment:
please also add

```
ipython
keras
h5py
```

It would have been great if this repo used poetry to make dep management a lot easier